api/src/server.ts

```diff
@@ -3,8 +3,25 @@ import cors from "cors";
 import * as notangles from "./notangles/index";
 import * as freerooms from "./freerooms/index";
 import { writeData } from "./write-data";
+import * as Sentry from "@sentry/node";
+import * as Tracing from "@sentry/tracing";
 
 const app = express();
+
+Sentry.init({
+    dsn: "https://d053cb7b3637458e974012782ee5f662@o1179870.ingest.sentry.io/6311064",
```

Comment:
Is this meant to be a secret? If so, can you figure out how to set it up/ask Jared?

Reply:
I gathered from the sentry documentation that the dsn is only used to tell sentry where to sen the error. So even though nobody can see any sensitive information with having the dsn, they can however use it to send errors to our account on sentry.
I read there is a way of choosing which IP address you want the errors to come from. So i guess we can specify the servers IP address for safety measures?

Reply:
Or another alternative to dsn is providing a url (which will be mentioned in the code as an option for Sentry.init()) which sentry can use to know where to capture error from.

Reply:
@lhjt this seems like it should be an env variable no?

Reply:
Is this URL also going to be on the frontend? In that case, it doesn't really matter, but I do think that setting it to an environment variable is a good idea anyway so that if the URL needs to be changed or something similar, I can just update the deployment configuration.